docs/plugins/lyrics.rst

```diff
@@ -113,3 +121,19 @@ After that, the lyrics plugin will fall back on other declared data sources.
 
 .. _pip: http://www.pip-installer.org/
 .. _BeautifulSoup: http://www.crummy.com/software/BeautifulSoup/bs4/doc/
+
+Activate on-the-fly translation
+-------------------------------
+
+Using the Bing Translation API requires `langdetect`_, which you can install
+using `pip`_ by typing::
+
+    pip install langdetect
+
+You also need to register for a Microsoft Azure Marketplace free account and
+to the `Microsoft Translator API`_.
+Set the ``bing_client_secret`` configuration option to your registered
+application password, alongside ``bing_lang_to`` target `language code`_.
```

Comment:
Perhaps we should include a few more instructions here about exactly how to get a client secret?